src/Microsoft.Extensions.FileProviders.Composite/CompositeFileProvider.cs

```diff
@@ -95,5 +96,10 @@ public IChangeToken Watch(string pattern)
             var CompositeFileChangeToken = new CompositeFileChangeToken(changeTokens);
             return CompositeFileChangeToken;
         }
+
+        /// <summary>
+        /// Gets the list of configured <see cref="IFileProvider" />.
+        /// </summary>
+        public IFileProvider[] FileProviders => _fileProviders;
```

Comment:
A lil late to party, perhaps `IReadOnlyList` in case we care about indexing into the sequence